public class TryCatchDemo {

	public static void main(String[] args) {
		int arr[] = new int[5];
		System.out.println(arr[10]);
		System.out.println("Line after exception");
	}
}
